.github/workflows/desktop-build.yaml

```diff
@@ -0,0 +1,110 @@
+name: Build Releases
+
+# This workflow builds the released desktop-version of naviz and drafts a new release.
+
+on:
+  push:
+    tags:
+      - "v*" # Build whenever a full version of naviz is released
+
+env:
+  CACHE_KEY: naviz-desktop
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        include:
+          - target: aarch64-apple-darwin
+            image: macos-latest
+          - target: x86_64-apple-darwin
+            image: macos-latest
+          - target: aarch64-unknown-linux-gnu
+            image: ubuntu-latest
+            prepare: sudo apt install -y gcc-aarch64-linux-gnu
+            env:
+              CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER: aarch64-linux-gnu-gcc
```

Comment:
absolutely no need to emulate here. GitHub offers native ubuntu ARM runners.

Reply:
That is interesting. I did not know that. Feel free to resolve the conversation in that case.